sentry_sdk/integrations/stdlib.py

```diff
@@ -79,12 +84,18 @@ def putrequest(self, method, url, *args, **kwargs):
                 url,
             )
 
+        sanitize = not _should_send_default_pii()
+        parsed_url = parse_url(real_url, sanitize=sanitize)
+
         span = hub.start_span(
-            op=OP.HTTP_CLIENT, description="%s %s" % (method, real_url)
+            op=OP.HTTP_CLIENT,
+            description="%s %s" % (method, parsed_url.url),
         )
 
         span.set_data("method", method)
-        span.set_data("url", real_url)
+        span.set_data("url", parsed_url.url)
+        span.set_data("http.query", parsed_url.query)
```

Comment:
I still think we should send `http.query` as a map/dict and not filter all the values out. 
Relay will currently take care of sensitive stuff in `span.data` if it finds something and later my scrubber can also be used to filter certain keys out.

cc: @cleptric

Reply:
fine with merging this for now I guess, don't wanna block or whatever
still think it's annoying to lose all query values as `FILTERED`

Reply:
I remember discussing that with @jjbayer and we agreed to send it as a string.
I opted to not scrub the data in `http.query` on the client, see https://github.com/getsentry/sentry-php/pull/1453.

Reply:
For now it is a string and filtered by default. I guess when we implement data scrubbers, we can change that.

Reply:
@antonpirker Just to make sure, even values in `data.http_query` are filtered?

Reply:
but like can we be consistent? I'm not sure what you guys agreed on but if Michi didn't do it and you do it here, it's already inconsistent.

Reply:
My understanding was that the whole idea was to move this data into a new property so server-side scrubbing can be applied more easily. If we also filter it by default in the SDKs, this feels like a regression to me.

Reply:
@cleptric Yes, everything in query is filtered by default. If you turn send_default_pii on, it will stay in there.

Reply:
But you are right @sl0thentr0py 
Just checked with the docs: https://develop.sentry.dev/sdk/data-handling/#structuring-data

It does not specify if all the values should be removed by default.
So the consistent way would be to ALWAYS remove `username:password` in the URL, but keep all the query parameters as is? (no matter what `send_default_pii` is set to, right?

Reply:
yes i think that's fine. We can move query params to span.data separately later.

Reply:
ty!

Reply:
We should move the query params to span data right now, this was again part of our initial idea. And @antonpirker is correct, the span descriptions should not contain any authority, query, or fragments anymore.
We can iterate on client-side filtering for the span data as a follow-up, happy to make changes to docs and PHP SDKs.

Reply:
ok then please do what Michi says :)

Reply:
done